static/styles/zulip.css

```diff
@@ -2336,6 +2336,7 @@ textarea.invitee_emails {
     max-height: 300px;
     width: 96%;
     max-width: 96%;
+    resize: vertical !important;
```

Comment:
What's this do?

Reply:
It makes the email text field be resized only vertically. Requested by Alya in [this comment](https://github.com/zulip/zulip/pull/23466#issuecomment-1422200962).